cmd/launcher/launcher.go

```diff
@@ -193,6 +193,9 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 	// we expect we're live. Record the version for osquery to
 	// pickup
 	internal.RecordLauncherVersion(rootDirectory)
+	if err = internal.RecordMetadata(rootDirectory, ctx, k); err != nil {
+		level.Error(logger).Log("msg", "unable to write metadata", "error", err.Error())
```

Comment:
we typically just use "err" as the key for error and don't need to add the `.String()`
```suggestion
		level.Error(logger).Log("msg", "unable to write metadata", "err", err)
```